### PR TITLE
UIDS-64 - Update Pill font weight and green bg color

### PR DESCRIPTION
**Pull request by fausgaitan**

### CODE REVIEW

**Review by davidmferris — APPROVED** [review]
Looks good!

Unrelated to this PR but I noticed our blue and orange pills are failing contrast accessibility checks, so might be something we want to think about updating:

Screen Shot 2020-10-23 at 7 38 58 AM

**Comment by fausgaitan** [comment]
> Looks good!
> 
> Unrelated to this PR but I noticed our blue and orange pills are failing contrast accessibility checks, so might be something we want to think about updating:
> 
> Screen Shot 2020-10-23 at 7 38 58 AM

I can definitely check this. I forgot I designed the others too and they should pass the check https://www.figma.com/file/471iS3QrpPtXZpS1xbsJ20/Components?node-id=1675%3A2

File: scss/colors/palette.scss [hunk 1]
```diff
@@ -9,7 +9,9 @@ $ux-orange: #F59C27;
 $ux-purple: #874B96;
 $ux-red: #FF4E00;
 $ux-white: #FFFFFF;
+
 $ux-yellow: #F6D810;
+$ux-yellow-100: #FFF6C6;
```
Comment (davidmferris) [inline scss/colors/palette.scss:14]: Nitpicking, but could we move `$ux-yellow-100` to the bottom of the file underneath all of the `$ux-red-` colors? I think we've tried to keep these alphabetized.

Comment (fausgaitan) [reply scss/colors/palette.scss]: done! 👍

**Review by davidmferris — APPROVED** [review]

**Review by rroppolo — APPROVED** [review]

**Pull request merged**